src/main/ipv-stub/ipv-authorize.ts

```diff
@@ -115,3 +119,27 @@ async function post(
     )
   );
 }
+
+function mapFormToUserIdentity(form: { [k: string]: string }): UserIdentity {
+  const coreIdentity = JSON.parse(form["identity_claim"]);
+  const addressClaim = JSON.parse(form["address_claim"]);
+  const drivingPermitClaim = JSON.parse(form["driving_permit_claim"]);
+  const socialSecurityRecordClaim = JSON.parse(form["nino_claim"]);
+  const passportClaim = JSON.parse(form["passport_claim"]);
+  const returnCodeClaim = JSON.parse(form["return_code_claim"]);
+  return {
+    sub: coreIdentity["sub"],
+    vot: coreIdentity["vot"],
+    vtm: coreIdentity["vtm"],
+    "https://vocab.account.gov.uk/v1/credentialJWT":
+      USER_IDENTITY["https://vocab.account.gov.uk/v1/credentialJWT"],
```

Comment:
When will we stop hard-coding this? Is there a ticket for it?

Reply:
Not yet - wanted to spend a bit of time in looking into this but got a placeholder. Hesitant to keep adding to the parent story as want to close it off soon.